tools/build

```diff
@@ -19,20 +19,23 @@ usage() {
     die 'build [ -n -pg1[234] ] ( test-crates | test-extension | test-post-install | test-doc | clippy )'
 }
 
-require_pgversion() {
-    [ -n "$pgversion" ] || die 'specify one of -pg12 | -pg13 | -pg14'
+require_pg_version() {
+    [ -n "$pg_version" ] || die 'specify one of -pg12 | -pg13 | -pg14'
 }
 
-find_pgconfig() {
-    require_pgversion
-    full_version=$(
-        cd "$HOME/.pgx"
-        set -- $pgversion.*/pgx-install
-        [ "$1" = "$pgversion.*/pgx-install" ] && die "$pgversion not installed in $HOME/.pgx"
-        set -- $(dirname $*)
-        [ $# -eq 1 ] || die "too many installations for $pgversion, found: $*"
-        echo $1)
-    pg_config="$HOME/.pgx/$full_version/pgx-install/bin/pg_config"
+find_pg_config() {
+    if [ -z "$pg_config" ]; then
```

Comment:
Here we add a check if pg_config is already specified before looking for it in ~/.pgx (which remains the default and what we use under the github action).